src/main/java/it/gov/pagopa/fdr/util/openapi/OpenAPIGenerator.java

```diff
@@ -34,12 +34,14 @@ public void filterOpenAPI(OpenAPI openAPI) {
 
     openAPI.setInfo(
         OASFactory.createInfo()
-            .title("FDR - Flussi di Rendicontazione")
+            .title("FDR - Flussi di Rendicontazione (${user_target})")
```

Comment:
Is the env variable declared here corrrect?

Reply:
Yes, it is used in `infra`-related TF for customize the JSON content. But it is just an additional information, it does not includes strange logic or other.